Library/Homebrew/test/rubocops/text/strict_spec.rb

```diff
@@ -131,5 +131,18 @@ def install
         end
       RUBY
     end
+
+    it 'reports an offense if "\#{bin}/<formula_name>" or other dashed binaries too are present' do
+      expect_offense(<<~RUBY, "/homebrew-core/Formula/foo.rb")
+        class Foo < Formula
+          test do
+            ohai "\#{bin}/foo", "-v"
+                 ^^^^^^^^^^^^ FormulaAuditStrict/Text: Use `bin/"foo"` instead of `"\#{bin}/foo"`
```

Comment:
Edge case to consider testing: `shell_output("#{bin}/foo --version")` where only a single argument is supported.